src/fields.jl

```diff
@@ -129,36 +145,44 @@ function 𝐼x!(output::Field2D{Face, Centre}, input::Field2D{Centre, Centre, Gr
         output.data[1, j] = (input.data[1, j] + input.data[nx, j]) / 2
     end
 
-    for j in 1:ny, i in 2:nx
+    for j in 1:ny, i in 1:nx
         output.data[i, j] = 𝐼xᶠᶜ(i, j, input)
     end
+
+    fill_halos!(output)
     
     return nothing
 end
 
-"""
-    𝐼x!(output::Field2D{Centre, Centre}, input::Field2D{Face, Centre, Grid2D{Periodic, Periodic}})
-
-Interpolates a 2D `input` field that lives on `Face`s `Centre`s to `output` field that lives on `Centre`s `Centre`s.
-"""
 function 𝐼x!(output::Field2D{Centre, Centre}, input::Field2D{Face, Centre, Grid2D{Periodic, Periodic}})
     nx, ny = input.grid.nx, input.grid.ny
     
     for j in 1:ny
         output.data[nx, j] = (input.data[1, j] + input.data[nx, j]) / 2
     end
```

Comment:
??

Reply:
This is still here...

Reply:
Oh, still something is left. I though I'd cleared up everything.